cfme/utils/appliance/__init__.py

```diff
@@ -1832,7 +1832,9 @@ def is_embedded_ansible_role_enabled(self):
 
     @property
     def is_embedded_ansible_running(self):
-        return self.is_embedded_ansible_role_enabled and self.supervisord.running
+        return self.is_embedded_ansible_role_enabled and (
+            (self.version < "5.11") == self.supervisord.running
```

Comment:
this statement looks weird please change it to something more readable

Reply:
@izapolsk Updated with previous logic, which much readable than current.

Reply:
thanks

Reply:
this old "new" logic looks much better but please pay attention to variable name next time.
your 'supervisord' variable retains process state not process itself. variable names should clearly describe their contents. I'd call it `process_state` or `is_running` or whatever.

and If I had to implement this logic, I'd write it in the following way 

```
return self.is_embedded_ansible_role_enabled and (self.version < '5.11' and self.supervisord.running)
```